.github/workflows/claude-dispatch.yml

```
@@ -0,0 +1,49 @@
+      # - name: Preliminary Setup
+        # run: |
+          # echo "Setting up environment..."
+          # Add any preliminary setup commands here to setup Claude's dev environment
+          # e.g., npm install, etc.
+
+      - name: Run Claude Code
+        id: claude
+        uses: anthropics/claude-code-action@eap
+        with:
+          mode: 'remote-agent'
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)